pkg/costmodel/costmodel.go

```diff
@@ -1445,7 +1447,7 @@ func requestKeyFor(startString string, endString string, windowString string, fi
 
 // Executes a range query for cost data
 func (cm *CostModel) ComputeCostDataRange(cli prometheusClient.Client, clientset kubernetes.Interface, cp costAnalyzerCloud.Provider,
-	startString, endString, windowString string, resolutionHours float64, filterNamespace string, filterCluster string, remoteEnabled bool) (map[string]*CostData, error) {
+	startString, endString, windowString string, resolutionHours float64, filterNamespace string, filterCluster string, remoteEnabled bool, offset string) (map[string]*CostData, error) {
```

Comment:
What should `offset` represent here? Just the Thanos offset, or is it a duplicate of `endString`? Since we already have `startString` and `endString` this is pretty confusing, so maybe at least a comment describing why we need both would be helpful.

Reply:
it's a duplicate of endstring, for convenience. I'll add a comment